src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -47,9 +42,15 @@ final class CacheItem implements CacheItemInterface
     /**
      * @param mixed $value
      */
-    public function __construct(string $key, $value, bool $isHit, ?ClockInterface $clock = null)
-    {
-        $this->key   = $key;
+    public function __construct(
+        /**
+         * Cache key
+         */
+        private string $key,
+        $value,
+        bool $isHit,
+        ?ClockInterface $clock = null
+    ) {
         $this->value = $isHit ? $value : null;
         $this->isHit = $isHit;
```

Comment:
Can perhaps be `private bool $isHit` in the ctor?

Reply:
updated at https://github.com/laminas/laminas-cache/pull/239/commits/0ae0e872b3052909299293c0752ca8f41a282f7f